IngresseSDK/Network/NetworkURLRequest.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  NetworkURLRequest.swift
+//  Alamofire
+//
+//  Created by Fernando Ferreira on 23/04/20.
```

Comment:
Geralmente a gente deixa assim no cabeçalho:

```suggestion
//
//  Copyright © 2020 ingresse. All rights reserved.
//
```